pkg/reconciler/events/cache/cacheclient.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cache
+
+import (
+	"context"
+
+	lru "github.com/hashicorp/golang-lru"
+	"k8s.io/client-go/rest"
+	"knative.dev/pkg/injection"
+	"knative.dev/pkg/logging"
+)
+
+// With 4 events per Run, we can store events for 1024 concurrent Runs
+const bufferSize = 4096
+
+func init() {
+	injection.Default.RegisterClient(withCacheClient)
+}
+
+// cacheKey is a way to associate the Cache from inside the context.Context
+type cacheKey struct{}
+
+func withCacheClientFromSize(ctx context.Context, size int) context.Context {
+	logger := logging.FromContext(ctx)
+
+	cacheClient, err := lru.New(size)
+	logger.Infof("CACHE CLIENT %+v", cacheClient)
```

Comment:
Is it required in the logs as info or was it a "debug" ? 😛

Reply:
Well, since this happens once per controller start-up, I thought it would be useful to have it at info to know that the cache was initialised.  If it should up a second time it means something is very wrong!